qa/remote-clusters/build.gradle

```diff
@@ -42,6 +44,7 @@ task copyKeystore(type: Sync) {
 elasticsearch_distributions {
   docker {
     type = 'docker'
+    architecture = Architecture.X64
```

Comment:
Should this technically use the _current_ architecture instead of being hard-coded to x86?

Reply:
Yes.